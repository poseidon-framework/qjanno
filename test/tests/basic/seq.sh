seq 1 1000 | qjanno --noHeader "SELECT avg(c1),sum(c1) FROM -"
