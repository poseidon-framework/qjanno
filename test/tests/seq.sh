seq 1 1000 | qhs "SELECT avg(c1),sum(c1) FROM -"
