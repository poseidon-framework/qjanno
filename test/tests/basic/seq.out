.---------.---------.
| avg(c1) | sum(c1) |
:=========:=========:
| 500.5   | 500500  |
'---------'---------'
